shared/src/main/scala/amf/client/remod/amfcore/config/AMFEventListener.scala

```diff
@@ -2,6 +2,6 @@ package amf.client.remod.amfcore.config
 
 import amf.core.model.document.BaseUnit
 
-private[remod] trait AMFEventListener {
+private[amf] trait AMFEventListener {
   def event(eventKind: String, bu: BaseUnit)
```

Comment:
We can take advantage of Scala's pattern matching to provide an `AmfEvent` instead of a simple string. The `AmfEvent` can hold much more information and could be usefull for the user (for example I imagine an approximated progress of the amount of document's parsed. That is not possible with just a plain string or at least will be limited to the formatting we give it.)